consensus/src/consensus/mod.rs

```diff
@@ -106,8 +132,17 @@ impl<N: Network> Consensus<N> {
     ) -> Self {
         let (tx, _rx) = broadcast(256);
 
+        //Setup the remote events handler
```

Comment:
You should add a linter script that finds these missing spaces :D 
```suggestion
        // Setup the remote events handler
```